### PR TITLE
Update setup.md - Round #1

### PR DESCRIPTION
**Pull request by Berealady9**
It has been a little while since I've worked on one of these. I tried to remember all the little formatting details, but I may have missed some things. I did make some broad changes to how things were structured a bit, so hopefully it still sounds ok.

### CODE REVIEW

File: sfcasts/setup.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+
+So what exactly *is* a service? That's easy! It's a plain PHP class that does *work*. For example, a `Logger` that helps you log messages is a service. Or a *mailer* that sends emails to your customers. *Or* a database connection object that you use to execute queries to the database. Those are *all* services. Even the controller that handles requests is a service, but it has *super powers*. We'll talk about that later.
+
+This course is titled "Fundamentals" because it's the *foundation*. Everything after this tutorial is just a variation of these themes. *So* to code along with me, download the course code on this page, unzip it, and inside, you'll find a `start/` directory with the same code that you see here. The `README.md` file has everything you need to get this application up and running. I've already completed most of these steps, so I'm going to move on to the last step and run the built-in Symfony web server. To do that, open your terminal and run:
+
+```terminal
```
Comment (MolloKhan) [inline sfcasts/setup.md:9]: @bocharsky-bw we use a special `terminal-silent` tag for those commands that you don't read in the video

Comment (bocharsky-bw) [reply sfcasts/setup.md:9]: yeah, right... ideally, we read all the commands, but during the code blocks adding if you notice that the command was not read - we should add change to `terminal-silent` then 👍

File: sfcasts/setup.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+
+Okay, so... how do we know which services we have? To see the list of all of the services we have available, we're going to use a special command. Over at your terminal, run:
+
+```terminal
+bin/console debug:container
+```
+
+Here you can see a long list of services we can use in our app. But where are these coming from? Who tells the container there should be a `logger` service whose class is `Logger` and it should be instantiated with these arguments? *Some* services are from our code and we'll talk about how those are registered in a bit. But the vast majority comes from bundles. Bundles are just plugins you can add to Symfony applications. They provide a few things, but the most important is services. Each bundle has a config file that says:
+
+`Hey! I want to have a service called "Logger"
```
Comment (bocharsky-bw) [inline sfcasts/setup.md]: We can make it as a blockquote using `>`

```suggestion
> Hey! I want to have a service called `logger`
```

File: sfcasts/setup.md [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+which should be an instance of "Logger", and it
```
Comment (bocharsky-bw) [inline sfcasts/setup.md]:
```suggestion
> which should be an instance of `Logger`, and it
```

File: sfcasts/setup.md [hunk 4]
```diff
@@ -1,3 +1,33 @@
...
+should be instantiated with these arguments.`
```
Comment (bocharsky-bw) [inline sfcasts/setup.md]:
```suggestion
> should be instantiated with these arguments.
```

**Review by bocharsky-bw — APPROVED** [review]
Thank you for fine-tuning it!

Not sure if Ryan will review it as well at this stage, but this looks good to me 👍

**Pull request merged**